test/integration/features/user_adds_template_to_project.spec.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+require('jasmine-beforeall');
+var h = require('../helpers');
+var projectHelpers = require('../helpers/project');
+var OverviewPage = require('../page-objects/overview').OverviewPage;
+var CreateProjectPage = require('../page-objects/createProject').CreateProjectPage;
+var DeploymentsPage = require('../page-objects/deployments').DeploymentsPage;
+var nodeMongoTemplate = require('../fixtures/nodejs-mongodb');
+//var logger = require('../helpers/logger');
+// TODO: use this to alter whitelist in the tests to support the create from url flow
+// var env = require('../helpers/env');
+
+describe('User adds a template to a project', function() {
+
+  beforeEach(function() {
+    h.commonSetup();
+    h.login();
+    projectHelpers.deleteAllProjects();
```

Comment:
Got it, I'll add the comment in the `projectHelpers` file above `deleteAllProjects`